main.go

```diff
@@ -243,17 +244,17 @@ func addPodSecurityPrivilegedLabels(watchNamespaceName string) error {
 		return err
 	}
 
-	privilegedLabels := map[string]string{
-		"pod-security.kubernetes.io/enforce": "privileged",
-		"pod-security.kubernetes.io/audit":   "privileged",
-		"pod-security.kubernetes.io/warn":    "privileged",
-	}
+	namespaceLabels := operatorNamespace.GetLabels()
+	// overwrite Pod security admission labels, if they exist; otherwise, add them
+	namespaceLabels["pod-security.kubernetes.io/enforce"] = "privileged"
```

Comment:
While we are at it, also convert the key and values to const string ?

Reply:
sure, will do in the next commit

Reply:
done